### PR TITLE
feat: Adding loki panel

### PR DESCRIPTION
**Pull request by markbeierl**
# Description

Changes the default time shown to 15 minutes
Expands all panels to be full screen width
Adds a loki log panel
Adds a variable for charm name to filter for the logs
Removes unused duplicate loki datasource 

# Checklist:

- [X] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
**N/A** I have added tests that validate the behaviour of the software
**N/A** I validated that new and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules
- [X] I have bumped the version of the library

### CODE REVIEW

File: grafana_dashboards/sdcore/5g_network.json
Comment (gruyaume) [inline grafana_dashboards/sdcore/5g_network.json:1]: There seems to be some lint check errors in this PR according to the CI logs: 

```
Checks that each panel uses the templated datasource.
Checks that each panel has at least one target.
Checks that each panel has a title and description.
Checks that each panel uses has valid units defined.
[❌] Dashboard '[5](https://github.com/canonical/sdcore-cos-configuration/actions/runs/8203224027/job/22435600768?pr=12#step:5:6)G Network Overview', panel 'UPF Status' has no or invalid units defined: ''
Checks that any counter metric (ending in _total) is aggregated with rate, irate, or increase.
Checks that every PromQL query has a instance matcher.
Checks that every PromQL query has a job matcher.
Checks that each target uses a valid PromQL query.
Checks that each target uses $__rate_interval.
Checks that the dashboard has a templated datasource.
[❌] Dashboard '5G Network Overview' templated data source variable named 'lokids', should be named 'loki_datasource'
[⚠️] Dashboard '5G Network Overview' templated data source variable labeled 'Loki datasource', should be labeled 'Loki data source'
[❌] Dashboard '5G Network Overview' templated data source variable named 'prometheusds', should be named 'prometheus_datasource'
[⚠️] Dashboard '5G Network Overview' templated data source variable labeled 'Prometheus datasource', should be labeled 'Prometheus data source'
Checks that the dashboard has a templated instance.
Checks that the dashboard has a templated job.
Checks that the dashboard templated labels have proper PromQL expressions.
Checks that the dashboard template variables are configured to reload on time change.
```

Note that the CI wasn't failing properly when lint checks fail. This PR addresses this:
- https://github.com/canonical/sdcore-cos-configuration/pull/14

Comment (gruyaume) [reply grafana_dashboards/sdcore/5g_network.json:1]: This is merged now, you should be able to rebase and see the linting errors here

**Review by Gmerold — COMMENTED** [review]
If we add logs to the existing dashboard, wouldn't it make it a bit crowded? 
Perhaps creating a separate dashboard for logs would be cleaner? We could also move both dashboards to a separate folder.
WDYT?

**Comment by markbeierl** [comment]
I am simply floating this as a concept. Can others give it a try and see if it is too crowded?

I am also not super happy with the log selector (Charm) being so far removed from the log panel. Eventually we will want to add selectors to see the charm logs too. There is a lot of opportunity for new ideas here, so I am going to park this PR until we can explore it more as a team.

**Comment by gatici** [comment]
https://github.com/canonical/sdcore-cos-configuration/pull/16 addresses this task. So, closing this PR.

**Pull request closed without merging**